lib/app/workorder.js

```diff
@@ -34,6 +36,20 @@ module.exports = function(mediator) {
     workorder.startTimestamp = newDate.getTime();
     workorder.finishTimestamp = tomorrow.getTime();
   });
-
-  return store.init('workorders', workorders, mediator);
+  
+  return store.init('workorders', workorders, mediator, function(dataStore, dataTopicPrefix) {
+    return dataStore.listen(dataTopicPrefix, mediator, {
+      // Override list method for workorders
+      list: function(filter) {
+        if (filter.sessionToken) {
+          return userSessionApi.getUserIdForSessionPromise(filter.sessionToken).then(function(userId) {
+            // Override filter
+            filter.value = userId;
```

Comment:
I think this is the `assignee` of the workorder instead of the value from the workorder data schema.

Also, the session filter value should be removed from the filter